src/python-common/ceph/deployment/drive_group.py

```diff
@@ -106,6 +106,22 @@ def __init__(self, msg):
         super(DriveGroupValidationError, self).__init__('Failed to validate Drive Group: ' + msg)
 
 
+class DriveGroupSpecs(object):
+
+    def __init__(self, drive_group_json: dict):
```

Comment:
We're still on py2 here.

Reply:
right, I was hoping that we've switched to py3 by the time I finished this PR :>